<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	background-color: #473C8B; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border-color: #8B7355;
	font: bold 14px;
	min-width:2em;
	color:white; /*字体颜色*/
	font-family:华文新魏;
	padding: 5px;
}
 
QPushButton:hover
{
	background-color: rgb(0, 150, 0);
}
 
QPushButton:pressed
{
	background-color: #1E90FF; /*伪状态经过时背景色*/ 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/10.png);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_push">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>私聊发送给:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>你的用户名:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索历史记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="room_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/3.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="your_id">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>textEdit</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>btn_push</zorder>
  <zorder>target_line</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>search1</zorder>
  <zorder>room_label</zorder>
  <zorder>your_id</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
